src/ActionList/Item.tsx

```diff
@@ -159,6 +159,14 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
         }
       },
 
+      '@media (forced-colors: active)': {
+        ':focus': {
+          // we set color to be transparent and let the high contrast rules
+          // decide what color with contrast should that be corrected to
+          outline: 'solid 1px transparent !important'
```

Comment:
Is there documentation about this browser behavior anywhere?

Reply:
Also what happens with safari with this media tag?

Reply:
> Is there documentation about this browser behavior anywhere?

Yep! Here's the [spec](https://www.w3.org/TR/css-color-adjust-1/#forced-colors-properties), [windows blog post](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/#:~:text=you%E2%80%99ll%20get%20a%20visual%20outline%20on%20the%20element). I got this trick from primer/css [action-list-item.scss](https://github.com/primer/css/blob/main/src/actionlist/action-list-item.scss#L56)

> Also what happens with safari with this media tag?

Safari doesn't support it, so it skips it. (Not sure if there's a way to make this work with windows high contrast + safari)